front/app/components/CustomFieldsForm/CustomFieldsPage.tsx

```diff
@@ -235,7 +242,13 @@ const CustomFieldsPage = ({
                         <AuthorField name="author_id" />
                       </Box>
                     )}
-
+                    {currentPageNumber === lastPageNumber - 1 &&
```

Comment:
This is a special field only visible to admins and only when the voting method is budgeting. I've created a separate component for it as it's not part of the custom fields list we get from the BE and it has special requirements.